viewer/src/components/Game.vue

```diff
@@ -390,42 +390,23 @@
                 </text>
             </template>
 
-            <g v-if="preferences.adjustPlayerOrder">
```

Comment:
![image](https://user-images.githubusercontent.com/342922/147362104-33383ae9-a59b-44b6-b457-664ad0cec152.png)

Ignoring whitespaces make this easier to review